docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -226,13 +226,49 @@ height={1618}
 
 When a partitioned asset depends on another partitioned asset, each partition in the downstream asset depends on a partition or multiple partitions in the upstream asset.
 
-A few rules govern partition-to-partition dependencies:
+A few rules govern default partition-to-partition dependencies:
 
 - When the upstream asset and downstream asset have the same <PyObject object="PartitionsDefinition" />, each partition in the downstream asset depends on the same partition in the upstream asset.
 - When the upstream asset and downstream asset are both time window-partitioned, each partition in the downstream asset depends on all partitions in the upstream asset that intersect its time window.
 
   For example, if an asset with a <PyObject object="DailyPartitionsDefinition" /> depends on an asset with an <PyObject object="HourlyPartitionsDefinition" />, then partition `2022-04-12` of the daily asset the would depend on 24 partitions of the hourly asset: `2022-04-12-00:00` through `2022-04-12-23:00`.
 
+#### PartitionMappings
+
+You can override these rules by providing a <PyObject object="PartitionMapping" /> when specifying a dependency on an asset. For example, here's how to specify that each partition of a daily-partitioned asset depends on the prior day's partition in an upstream asset:
```

Comment:
> You can override these rules by providing

In this sentence, `these` is an unclear antecedent - it's not immediately clear what `these` refers to. I think that's partially because this sentence starts a new section, so it may take a moment to connect the dots.

Consider something like 

- You can override the default partition dependency rules by... 
- To override the default partition dependency rules, provide a...

I know these suggestions are a bit wordy, but it's clear what it's referring to.